pkg/kritis/metadata/grafeas/grafeas.go

```diff
@@ -232,6 +232,21 @@ func (c Client) CreateAttestationOccurence(note *grafeas.Note,
 	return c.client.CreateOccurrence(c.ctx, req)
 }
 
+// Builds gets Build Occurrences for a specified image.
+func (c Client) Builds(containerImage string) ([]metadata.Build, error) {
+	occs, err := c.fetchOccurrence(containerImage, "BUILD")
+	if err != nil {
+		return nil, err
+	}
+	var builds []metadata.Build
+	for _, occ := range occs {
+		if v := util.GetBuildFromOccurrence(occ); v != nil {
+			builds = append(builds, *v)
+		}
+	}
+	return builds, nil
```

Comment:
NIT: Add `glog.Infof("got build occurrences (%d) for %s", len(builds), containerImage)` here too. (Although, we will not be using this feature)

Reply:
Fixed in 8499f80